newapi/debian/ooni_download_geoip.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+TS=$(date +"%Y-%m")
+
+CURRENT_TS=$(cat /var/lib/ooniapi/geoipdbts)
+if [ "$CURRENT_TS" = "$TS" ];then
+    echo "Already the latest. Skipping"
+    exit 0
+fi
+
+ASN_URL="https://download.db-ip.com/free/dbip-asn-lite-${TS}.mmdb.gz"
+CC_URL="https://download.db-ip.com/free/dbip-country-lite-${TS}.mmdb.gz"
+
+ASN_STATUS=$(curl -s -o /dev/null -I -w "%{http_code}" ${ASN_URL})
+if [ "$ASN_STATUS" != "200" ];then
+    echo "$ASN_URL hasn't been updated yet"
+    echo "$ASN_URL hasn't been updated yet" | systemd-cat -t ooniapi-geoip
+    # We exit with 0 to avoid systemd thinking the unit has failed
+    exit 0
+fi
+
+CC_STATUS=$(curl -s -o /dev/null -I -w "%{http_code}" ${CC_URL})
+if [ "$CC_STATUS" != "200" ];then
+    echo "$CC_URL hasn't been updated yet"
+    echo "$CC_URL hasn't been updated yet" | systemd-cat -t ooniapi-geoip
+    # We exit with 0 to avoid systemd thinking the unit has failed
+    exit 0
+fi
+
+mkdir -p /var/lib/ooniapi/
+
+echo "Updating GeoIP database from $CC_URL $ASN_URL" | systemd-cat -t ooniapi-geoip
+curl -o /var/lib/ooniapi/asn.mmdb.gz ${ASN_URL}
+gunzip /var/lib/ooniapi/asn.mmdb.gz
+curl -o /var/lib/ooniapi/cc.mmdb.gz ${CC_URL}
+gunzip /var/lib/ooniapi/cc.mmdb.gz
+
+# Basic smoke test of the geoip databases
+python <<EOF
```

Comment:
We should rather switch the rest of the script to Python so it's all the same language. As an added benefit we get to remove the dependency on curl.

Reply:
Ok done